syntax = "proto3";

service OrderManagement {
	rpc CreateOrder(Order) returns (OrderResponse);
}

message Order {
	string product_name = 1;
	float price = 2; 
	string payment_gateway = 3; 
	string card_brand = 4; 
	string bank = 5;
	string region = 6;
	string address = 7; 
	string email = 8; 
}

message OrderResponse {
	string status = 1;
	string order_id = 2;
}

